### ISSUE TITLE
Pull Request

### ISSUE THREAD

**Issue by impeeza** [mrdude2478/Tinwoo-Release#3]
Hello  Dude,  please take a look on https://github.com/impeeza/TinWoo/pull/1  regards.

**Comment by mrdude2478** [comment]
@impeeza 

Yep, seen it, I've implemented it this morning but not tested yet as I've been busy all day. I've just got to add a setting in the menu to disable the tik check warning. Also I need to take a look at some issue that was reported with usb losing connection and needing a restart. I'm a bit busy at the moment but if I get a chance I'll push a new build sometime in the next few days.

**Comment by mrdude2478** [comment]
Closing now as patch added to the new release. Thanks for pointing this out (I'd already seen it). If You have anymore patches you wish to be implementeed let me know.
